config/default/core.entity_form_display.node.news.default.yml

```diff
@@ -132,19 +125,12 @@ content:
     weight: 5
     region: content
     settings:
-      title: Quote
-      title_plural: Quotes
+      title: Paragraph
```

Comment:
We probably don't need all of these changes.

Reply:
reverted